### DISCUSSION TITLE
Practica 1 - Lo que llevo

### DISCUSSION THREAD

**Discussion by ElPoder1** [Launch-X-Latam/MisionFrontEnd#144] [Show and tell]
Buen día, les comparto lo que a mi entender son las practicas con la finalidad que sirvan de retroalimentación y aceptar comentarios y/o sugerencias por si a alguien mas le sirve también.

Las practicas las estoy colocando en un repositorio que creé y se los comparto en un link continuación, aunque no se si asi se vayan a entregar jaja, la ultima parte de UI aun la estoy haciendo en Figma pero ahí la llevo.

**[Link de Abogabot en mi repositorio](https://github.com/ElPoder1/Launch-X-Projects/blob/main/Frontend%20Practicas/Abogabot/1-Abogabot.md)**

Salu2

**Comment by ghost** [comment]
muy bueno!
